wordpress-mobile/WordPress-iOS
# Implement content info metrics (word, characters) in Gutenberg.

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/506

GB-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2380

<img src="https://user-images.githubusercontent.com/651601/84710175-9bdc1d00-af5b-11ea-8a68-404c836ba10c.png" width=300>

To test:
 - Start the app
 - Open a post using the Block editor
 - Tap on the ... button on the top right
 - Check if the post metrics are displayed on the top of action sheet
 

PR submission checklist:
